docker/Dockerfile

```diff
@@ -279,42 +268,44 @@ ENV VLLM_TARGET_DEVICE=${vllm_target_device}
 # Skip adding +precompiled suffix to version (preserves git-derived version)
 ENV VLLM_SKIP_PRECOMPILED_VERSION_SUFFIX=1
 
-RUN --mount=type=cache,target=/root/.cache/uv \
-    --mount=type=bind,source=.git,target=.git \
+RUN --mount=type=bind,source=.git,target=.git \
```

Comment:
Since .git causes the docker build cache to misses everything from this point on.